command/agent/job_endpoint.go

```diff
@@ -362,16 +363,17 @@ func (s *HTTPServer) jobActions(resp http.ResponseWriter, req *http.Request, job
 	return out.Actions, nil
 }
 
-func (s *HTTPServer) jobRunAction(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
-
-	s.logger.Info("jobRunAction called")
-
-	// Build the request and parse the ACL token
+func (s *HTTPServer) jobRunAction(resp http.ResponseWriter, req *http.Request, jobID string) (interface{}, error) {
 	task := req.URL.Query().Get("task")
 	action := req.URL.Query().Get("action")
 	allocID := req.URL.Query().Get("allocID")
+
+	s.logger.Info("job action called",
```

Comment:
Possibly worth noting that the existing `alloc exec` doesn't have this log line.

Reply:
This had been a debugging-purposes log for me at first and I thought it might still be relevant as a server log, but it probably doesn't meet the "more signal than noise" threshold.

Reply:
Hum....yeah, and I think we log every HTTP request so, to some degree we already have a log for when this happens? I think I will remove this.